index.html.in

```diff
@@ -125,6 +125,9 @@
                           </div>
                           <div class="form-group hidden-xs">
                               <div class="col-sm-6">
+                                <button type="button" class="btn btn-sm btn-default pull-left" id="cancelWebpageTranslate" style="    display: none;">
```

Comment:
There was a newline there in sublime text and the space wasn't visible :P. Modified :D